Vagrantfile

```diff
@@ -19,5 +19,7 @@ install podman make golang-go libseccomp-devel bats jq
 ts run
 EOF
     dnf clean all
+    # Add a user for rootless tests
+    useradd -u2000 -m -d/home/rootless -s/bin/bash rootless
```

Comment:
BTW I am doing this (and more) in https://github.com/opencontainers/runc/pull/2342

Reply:
yes, copied from your PR :)